infra/base-images/base-runner/Dockerfile

```diff
@@ -14,11 +14,42 @@
 #
 ################################################################################
 
+# Build go stuff in its own image. We only need the resulting binaries.
+# Keeping the go toolchain in the image wastes 400 MB.
+FROM gcr.io/oss-fuzz-base/base-image as go-builder
+
+RUN apt-get update && apt-get install wget -y
+
+# Download and install the latest stable Go.
```

Comment:
Why are we doing this ? arent we copying stuff from go-builder ?

Reply:
Right. So go-builder is one image that we install the go toolchain and then use it to build the golang binaries we need.
Then in gcr.io/oss-fuzz-base/base-image we copy those binaries without the toolchain (which is about 400MB of unnecessary space).